pkg/commands/edit/remove/util.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package remove
+
+import (
+	"log"
+	"sigs.k8s.io/kustomize/v3/pkg/fs"
+)
+
+func globPatternsFS(fsys fs.FileSystem, patterns []string) ([]string, error) {
```

Comment:
can we reuse ./pkg/commands/edit/add/util.go ?

maybe move it up a directory into the `edit` package and make it public?

Reply:
Fixed in d34c82c and 5c93722